pkg/config/config.go

```diff
@@ -111,7 +112,7 @@ func initializeFlags() *EnvironmentFlags {
 
 	defaultCatalogSource := os.Getenv("CATALOG_SOURCE")
 	flag.StringVar(&f.CatalogSource, "catalogsource", defaultCatalogSource,
-		"Provide defaultCatalogSource to subscribe operator from. By default `pre-stage-operators` will be used.")
+		"Provide defaultCatalogSource to subscribe operator from. By default `custom-operators` will be used.")
```

Comment:
It's not related to this PR but are we using these strings/descriptions anywhere? Most of them are outdated or misleading.